docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -107,20 +75,35 @@ spec:
       vlan: 22
 EOF
 ----
++
+* In case when `neutron-sriov-nic-agent` is running on the existing Compute nodes, physical device mappings needs to be checked and set the same in the
+`OpenStackDataPlaneNodeSet` custom resource (CR). Those options will need to be set in the `OpenStackDataPlaneNodeSet` CR. For more information, see xref:reviewing-the-openstack-control-plane-configuration_adopt-control-plane[Reviewing the {rhos_prev_long} control plane configuration].
+
+.Variables
```

Comment:
So, is *Variables* going to be a "convention" used in this guide?

I would be tempted to make *Variables* a bullet item under prerequisites:

````
- Define the shell variables necessary to run the script that stops the
  infrastructure management and the compute services.
  
  <code snippet>
  
  o Replace <path to SSH key> with the absolute path to the SSH key
     for the blah, blah, blah.
  o Replace standalone.localdomain with the IP address for your
      local domain.
````